ChangeLog

```diff
@@ -9,6 +9,11 @@ Release date: TBA
 ..
   Put new features here and also in 'doc/whatsnew/2.13.rst'
 
+* Add ``iterating-modified-list`` checker for lists which items are added or removed
+  during iteration while they are iterating through.
```

Comment:
```suggestion
* Add ``iterating-modified-list``, emitted when items are added or removed to a list being iterated through.
```

Reply:
The other messages should also be listed here and in the other changelog

Reply:
Although they're not implemented yet?

Reply:
Oh sorry, I didn't see that before writing this.

Do you plan on adding them as well in this PR? If not, you should probably also remove their messages and only add those in the relevant PRs

Reply:
@DanielNoord  Just added them :)